### PR TITLE
Added evaluation directory

### PR DESCRIPTION
**Pull request by luisenp**

### CODE REVIEW

File: examples/configs/tactile_pose_estimation.yaml [hunk 1]
```diff
@@ -44,7 +51,8 @@ tactile_cost:
 
 options:
   vis_traj: True
+  vis_train: true
 
 hydra:
   run: 
-    dir: examples/outputs
\ No newline at end of file
+    dir: examples/outputs/tactile
```
Comment (mhmukadam) [inline examples/configs/tactile_pose_estimation.yaml:58]: missing new line

File: experiments/evaluation/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+This folder contains scripts for evaluating `theseus`'s components.
```
Comment (mhmukadam) [inline experiments/evaluation/README.md:1]: Nice! Maybe also add a line about what each .sh script does and if there are any params involved in running the scripts.

Comment (mhmukadam) [reply experiments/evaluation/README.md:1]: Oh I see each script has starting comment with this info. Might still be useful to have in the readme though.

**Review by mhmukadam — APPROVED** [review]

File: experiments/evaluation/README.md (hunk 2 above)
Comment (luisenp) [reply experiments/evaluation/README.md:1]: Added. @fantaosha can you confirm the descriptions for PGO look OK?

**Review by fantaosha — APPROVED** [review]
Removed `pose_graph_g2o`.

**Pull request merged**